lib/warder/rails_advice_runner.rb

```diff
@@ -5,13 +5,20 @@ class RailsAdviceRunner < Runner
     CLI_FULL_OPTION = 'rails-advice'
     DESCRIPTION = 'Run rails best practices validation'
     COMMAND_NAME = 'rails_best_practices'
-    FAILURE_REGEXP = /Found (\d+) warnings?/
+    COMMAND_OPTIONS = '--without-color --silent --spec --test --features'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.